pdf/src/campPrint/RichText.vue

```diff
@@ -70,6 +70,23 @@ const rules = [
   {
     shouldProcessNode: (node) => node.type === 'tag' && node.name === 'li',
     processNode: (node, parent) => {
+      const childWithEmptyText = {
+        type: 'text',
+        name: '',
+        attrs: {},
+        children: [],
+        content: '',
+      }
+      if (!node.children.length) {
+        node.children.push({
+          type: 'text',
+          attrs: {},
+          children: [childWithEmptyText],
+        })
+      }
+      if (!node.children[0].children.length) {
+        node.children[0].children.push(childWithEmptyText)
+      }
```

Comment:
My proposal to make it more readable (untested):
```suggestion
      const emptyChild = {
        type: 'text',
        attrs: {},
        children: [],
      }
      if (!node.children.length) {
        node.children.push({ ...emptyChild })
      }
      if (!node.children[0].children.length) {
        node.children[0].children.push({ ...emptyChild, content: '' })
      }
```